src/menu/menu.scss

```diff
@@ -149,6 +150,32 @@ $iui-active-outline: thin solid t(iui-color-foreground-primary);
       }
     }
   }
+
+  &-skeleton {
```

Comment:
I was thinking whether to create completely separate class (not overriding existing css) but it doesn't have a lot of css and overrides so just extended the existing menu item css.